editoast/src/models/stdcm_log.rs

```diff
@@ -25,3 +30,37 @@ pub struct StdcmLog {
     pub created: DateTime<Utc>,
     pub user_id: Option<i64>,
 }
+
+impl StdcmLog {
+    // We just don't await the creation of the log entry since we want
+    // the endpoint to return as soon as possible, and because failing
+    // to persist a log entry is not a very important error here.
```

Comment:
Shouldn't this comment stay near the `span`? It mentions an "endpoint" and awaiting or not the return value is a decision for the caller.

Reply:
Good point.